torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -88,8 +89,21 @@ def validate_args_and_maybe_create_graph_inputs(
             if manually_set_subgraph_inputs:
                 new_proxy = tracer.create_graph_input(a.as_proxy().node.name)
                 example_value = a.as_proxy().node.meta["example_value"]
+
+                options = {}
+                if is_traceable_wrapper_subclass(example_value):
+                    # Note: this is a temporary hack to unblock a single tensor
+                    # subclass use case. Passing these options in the general
+                    # case breaks various tests. We should fix it more properly
+                    # in the future.
+                    options = VariableTracker.propagate(a)
+                    options["source"] = LocalSource(a.as_proxy().node.name)
```

Comment:
Our little VC source creation hack was just to unblock, this won't fly in real life :(